paragon/_overrides.scss

```diff
@@ -0,0 +1,149 @@
+  &.site-header-desktop {
+    background-color: $header-bg;
+  }
+
+  @if variable-exists(customize-header-border) {
+    // Set the top and bottom border of the header
+    border-top: $header-border-top-width solid $header-border-top-color;
+    border-bottom: $header-border-bottom-width solid $header-border-bottom-color;
+    // The default height is 3.75rem, to that we add the additional border size, otherwise
+    // it breaks the UI.
+    height: calc(3.75rem + #{$header-border-top-width + $header-border-bottom-width});
+  }
+  // Set the color of text in the header - course about page.
+  @if variable-exists(main-nav-color) {
+    span {
+      color: $main-nav-color;
+    }
+  }
+
+  .nav-links {
+    @if variable-exists(main-nav-link-color) {
+      // Set the color of the links in the dashboard
+      .main .nav-tab a,
+      .secondary .nav-item a,
+      .secondary .nav-item a span,
+      .toggle-user-dropdown span {
+        color: $main-nav-link-color !important;
+      }
+    }
+
+    @if variable-exists(main-nav-item-border-bottom-color) {
+      // Set the bottom border color of the links in the dashboard like 'Courses', 'Programs', 'Profile' etc.
+      .main .nav-tab a {
+        &.active {
+          border-bottom-color: $main-nav-item-border-bottom-color !important;
+          border-bottom-style: solid !important;
+        }
+      }
+    }
+
+    @if variable-exists(main-nav-item-hover-border-bottom-color) {
+      .main .nav-tab a {
+        &:not(.active):hover {
+          border-bottom-color: $main-nav-item-hover-border-bottom-color !important;
+          border-bottom-style: solid !important;
+        }
+      }
+    }
+
+    @if variable-exists(user-dropdown-color) {
+      // Customize the color of the links in the user dropdown menu
+      .secondary .dropdown-user-menu .dropdown-item a {
+        color: $user-dropdown-color !important;
+      }
+    }
+  }
+
+  .main .nav-tab a,
+  .secondary .nav-item a:not(.btn):not(.help-link),
+  .secondary .nav-item .toggle-user-dropdown {
+    // Set the color of the dropdown arrow of the user dropdowm menu
+    color: $link-color !important;
+  }
+
+
+    @if variable-exists(customize-sign-in-btn) {
+      // Customize the sign-in button on the home page header
+      a.sign-in-btn {
+        @include btn-styles(
+          $btn-sign-in-bg,
+          $btn-sign-in-color,
+          $btn-sign-in-border-color,
+          $btn-sign-in-hover-bg,
+          $btn-sign-in-hover-color,
+          $btn-sign-in-hover-border-color
+        );
+      }
+
+    @if variable-exists(customize-register-btn) {
+      // Customize the register button on the home page header
+      a.register-btn {
+        @include btn-styles(
+          $btn-register-bg,
+          $btn-register-color,
+          $btn-register-border-color,
+          $btn-register-hover-bg,
+          $btn-register-hover-color,
+          $btn-register-hover-border-color
+        );
+      }
+    }
+  }
```

Comment:
Most of these rules are identical to `theme-overrides`. Could we create a mixin in common overrides to avoid duplicating code?

Also, I see that some of these might not be applicable to the existing MFEs (or at least I cannot find them being used) - e.g. sign-in button, register button, nav button's border, dropdown items (mentioned in a comment above).

Reply:
Yup. The idea is to move all common code and mixins to `/common/_overrides.scs`. So the structure is what I plan to use. There may be more files created in `/common` for fonts etc, but the idea is that they are used across platforms. 

However, that level of optimisation is not in scope currently. As the new theme is developed this code might be rewritten to adapt to it.